packages/blocks/tsconfig.json

```diff
@@ -5,6 +5,7 @@
   ],
   "compilerOptions": {
     "types": ["svelte", "vite/client", "vitest"],
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
Do we have a ticket to track moving this into `js-config`? Do we want that in `js-config`?

Reply:
We do have one! Ticket here: https://viam.atlassian.net/browse/APP-2482.

I think the best course of action is to disable until we can fix at the source